hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyMetrics.java

```diff
@@ -44,7 +44,12 @@ public final class AMRMProxyMetrics {
   @Metric("# of failed allocate requests ")
   private MutableGaugeLong failedAllocateRequests;
   @Metric("# of failed application recoveries")
-  private MutableGaugeLong failedAppRecoveryCount;
```

Comment:
I think we need to keep backwards compatibility; we cannot just remove an existing metric.

Reply:
I will fix it.